Express-API/roles.js

```diff
@@ -0,0 +1,23 @@
+const createRoleQuery = "INSERT INTO role(name, department_id, band_id, summary, job_spec_url) VALUES (?,?,?,?,?) "
+const createRoleCapabilityLinkQuery = "INSERT INTO role_capability(role_id,capability_id) VALUES (?,?) "
+
+class RolesHandler {
+    constructor( config ) { }
+
+    createRole(params, db){
+        return new Promise( ( resolve, reject ) => {
+            try {
+                db.query(createRoleQuery,[params.roleName,params.departmentId,params.bandId,params.summary,params.jobSpecUrl]).then(rows => {
+                    console.log(rows.insertId)
```

Comment:
Can this comment be removed?

Reply:
Yeah I'd definitely remove that log here